<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Tab1SocketClient</class>
 <widget class="QWidget" name="Tab1SocketClient">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="1,8,1">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="3,1,1">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string> 수신 데이터</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pPBRecvDataClear">
       <property name="text">
        <string>수신 삭제</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pPBServerConnect">
       <property name="text">
        <string>서버 연결</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QTextEdit" name="pTERecvData">
       <property name="overwriteMode">
        <bool>true</bool>
       </property>
       <property name="tabStopWidth">
        <number>800</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,3,1">
     <item>
      <widget class="QLineEdit" name="pLERecvId"/>
     </item>
     <item>
      <widget class="QLineEdit" name="pLESendData"/>
     </item>
     <item>
      <widget class="QPushButton" name="pPBSendButton">
       <property name="text">
        <string> 송신</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
